packages/datadog-plugin-moleculer/src/client.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const { moleculerTags } = require('./util')
+
+function createWrapCall (tracer, config) {
+  return function wrapCall (call) {
+    return function callWithTrace (actionName, params, opts) {
+      const options = {
+        service: config.service,
+        resource: actionName,
+        tags: {
+          'span.kind': 'client'
+        }
+      }
+
+      opts = arguments[2] = opts || {}
+      opts.meta = opts.meta || {}
+
+      arguments.length = Math.max(3, arguments.length)
```

Comment:
Why change the args length? In general, we always want to preserve args, no?

Reply:
Otherwise even if `opts` is changed, it's ignored because the arguments object can be too short.

Reply:
SGTM